### PR TITLE
Generate PO file from Inno setup isl file.

### PR DESCRIPTION
**Pull request by mrpau-richard**
### Summary 
I created a Python script to create po file and extract translatable strings from Inno setup isl file.

### Issue addressed 
#29 

### TODO

- [x] Create to script to encode translated strings from po file to inno setup isl file.

/cc @radinamatic

### CODE REVIEW

**Comment by mrpau-richard** [comment]
@radinamatic here's the [installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMZXJCMlkyUjBNTkk) to test the Spanish language translated from the crowdin.

I'm currently working on the documentation on how to extract and encode the strings from the Inno setup `isl` file.

**Comment by radinamatic** [comment]
@mrpau-richard I'm going through the installer steps to see how can I improve the translation, but the main problem I see for now is codification: Spanish accented vowels do not display correctly:

fullscreen_04_10_17_02_42

fullscreen_04_10_17_02_45

**Comment by mrpau-richard** [comment]
@radinamatic I think that's an installer issue, let me fix that one.

**Comment by mrpau-richard** [comment]
@radinamatic I've fixed the Spanish text that's not properly displaying. I also added some languages that's been translated in [Inno setup official translations](http://www.jrsoftware.org/files/istrans/).
Here's a sample screenshots:
screen shot 2017-10-04 at 1 03 59 pm
screen shot 2017-10-04 at 1 03 39 pm

Here's the [installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMWi1GWThMeGc4eFU) for your testing.

**Comment by radinamatic** [comment]
@mrpau-richard Ok, looking much better now, codification issues gone! :+1: 

On another hand, there are still a couple of problems:

1. Button strings in this window are not localized:

fullscreen_04_10_17_12_13

2. This installer is not merged with the latest enhancements, right? Since I can still see the steps that we made invisible to the user:

fullscreen_04_10_17_12_17

3. After the installation is done in Spanish, for example, the startup messages should be in that language  too. Right now they still appear in English:

fullscreen_04_10_17_12_20

fullscreen_04_10_17_12_21

4. If the language used in the installation up to this point is present in Kolibri, the ideal would be to have it selected by default on this step too, although I'm not sure if you can pass that argument to the installer... :thinking: 

fullscreen_04_10_17_12_34

Maybe @indirectlylit and @rtibbles could chime on this too.

**Comment by mrpau-richard** [comment]
@radinamatic the reason why those message box buttons is in english it's because your Windows language is in English. Here's the reference http://www.jrsoftware.org/iskb.php?translatebuttons

**Comment by radinamatic** [comment]
Makes sense, @mrpau-richard!
One thing less to fix :+1:

**Comment by mrpau-richard** [comment]
@radinamatic We can also use the selected language from the installer into GUI application and Kolibri itself. we just need to set the selected language into environment variable and we can use it in Kolibri by running this command `kolibri language setdefault lang` before starting the server.

File: windows/installer-source/KolibriSetupScript.iss [hunk 1]
```diff
@@ -31,11 +31,19 @@ UsePreviousAppDir=yes
 ChangesEnvironment=yes
 
 [Languages]
-Name: "English"; MessagesFile: "compiler:default.isl"
+Name: "english"; MessagesFile: "compiler:Default.isl"
+Name: "spanish"; MessagesFile: "compiler:Languages\Spanish.isl"
```
Comment (indirectlylit) [inline windows/installer-source/KolibriSetupScript.iss]: curious why `english` and `spanish` are lowercase but the other languages are uppercase?

**Comment by indirectlylit** [comment]
mind resolving merge conflicts and posting a new installer to test? thanks!

**Comment by mrpau-richard** [comment]
@radinamatic & @indirectlylit here's the newly built [installer](https://drive.google.com/open?id=0B5Tyi-tRKOFMU1Njd0VnME5tSUE) to test this PR with the update's from the develop branch.

**Comment by indirectlylit** [comment]
looks like a (very minor) change to the readme got reverted in the merge:

image

no worries, I'll re-make the change

**Pull request merged**